### PR TITLE
ISIS template text separator in mobile view

### PR DESCRIPTION
**Pull request by brianteeman**
PR for #16267

### Before
screenshotr14-13-00

### After
screenshotr14-07-34

### CODE REVIEW

File: media/jui/less/responsive-767px-max.less [hunk 1]
```diff
@@ -37,6 +37,14 @@
     }
   }
 
+  // MENU SEPERATOR
```
Comment (Quy) [inline media/jui/less/responsive-767px-max.less]: Spelling

File: media/jui/less/responsive-767px-max.less [hunk 2]
```diff
@@ -37,6 +37,14 @@
...
+  // --------------
+  // Reset background
+  .dropdown-menu .menuitem-group {
+    background-color:#10223e;
```
Comment (Quy) [inline media/jui/less/responsive-767px-max.less]: Add space after `:`

File: media/jui/less/responsive-767px-max.less (hunk 1 above)
Comment (brianteeman) [reply media/jui/less/responsive-767px-max.less]: fiddlesticks - i did it again

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 0afa476984d19f1ed1874d8a9116dd09f64d9189<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18225">issues.joomla.org/tracker/joomla-cms/18225</a>.</sub>

**Comment by ghost** [comment]
This is an easy Test. Needs a second successfully Test.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18225">issues.joomla.org/tracker/joomla-cms/18225</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 0afa476984d19f1ed1874d8a9116dd09f64d9189<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18225">issues.joomla.org/tracker/joomla-cms/18225</a>.</sub>

**Comment by ghost** [comment]
RTC after two successful tests.

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**